rest/route/subscription_route_test.go

```diff
@@ -0,0 +1,83 @@
+package route
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+	"testing"
+
+	"github.com/evergreen-ci/evergreen"
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/evergreen-ci/evergreen/model/event"
+	"github.com/evergreen-ci/evergreen/model/user"
+	"github.com/evergreen-ci/evergreen/rest/data"
+	"github.com/evergreen-ci/evergreen/testutil"
+	"github.com/stretchr/testify/suite"
+)
+
+type SubscriptionRouteSuite struct {
+	sc data.Connector
+	suite.Suite
+	postHandler MethodHandler
+}
+
+func TestSubscriptionRouteSuiteWithDB(t *testing.T) {
+	s := new(SubscriptionRouteSuite)
+	db.SetGlobalSessionProvider(testutil.TestConfig().SessionFactory())
+	s.sc = &data.DBConnector{}
+
+	// run the rest of the tests
```

Comment:
the rest?

Reply:
oops, that comment was copied from elsewhere

Reply:
I think you didn't delete it in your next patch.